python/mlcroissant/mlcroissant/_src/core/json_ld.py

```diff
@@ -114,6 +114,20 @@ def unbox_singleton_list(d: Any):
     return d
 
 
+def box_singleton_list(element: Any) -> list[Any] | None:
+    """Returns the elemtn always boxed inside a list.
```

Comment:
element

inside a list -> inside of a list

Can you also add an example of such property? I assume things such as containedIn woulf fix into this box?

Reply:
Done. The docstring is better now, thanks!